docs/src/lib/indexing.md

```diff
@@ -25,13 +25,13 @@ The rules for a valid type of index into a column are the following:
       `AbstractVector{Symbol}`);
     * a vector of `AbstractString` (does not have to be a subtype of
       `AbstractVector{<:AbstractString}`);
-    * a vector of `Integer` other than `Bool` (does not have to be a subtype of
+    * a vector of `Integer` that are not `Bool` (does not have to be a subtype of
       `AbstractVector{<:Integer}`);
-    * a vector of `Bool` that has to be a subtype of `AbstractVector{Bool}`;
-    * a regular expression, which gets expanded to a vector of matching column
-      names;
+    * a vector of `Bool` (must be a subtype of `AbstractVector{Bool}`);
+    * a [regular expression](https://docs.julialang.org/en/v1/manual/strings/#Regular-Expressions) (will be expanded to a vector of matching column
```

Comment:
Any way to make this link more future-proof?

Reply:
I think it is OK